dir.props

```diff
@@ -103,7 +103,8 @@
     <NugetRestoreCommand>$(NugetRestoreCommand) install</NugetRestoreCommand>
     <!-- Trim off the last slash so that nuget doesn't get confused and say there are illegal characters on
          the path. -->
-    <NugetRestoreCommand>$(NugetRestoreCommand) -OutputDirectory "$(PackagesDir.TrimEnd('\\'))"</NugetRestoreCommand>
+    <NugetRestoreCommand Condition="'$(OsEnvironment)'=='Windows_NT'">$(NugetRestoreCommand) -OutputDirectory "$(PackagesDir.TrimEnd('\\'))"</NugetRestoreCommand>
```

Comment:
Where do you guys use NugetRestoreCommand? Hopefully no where and this can just be deleted.